python/README.rst

```diff
@@ -58,6 +66,244 @@ Installation
+coordinates):
+
+1. `orbit id`: an integer number to identify the orbit; usually ranges
+   from 0 to `n`-1, where `n` is the number of orbits
+2. `perihelion distance` (au) for ``COM``, semimajor axis `a` (au) for
+   ``KEP``, `x` (au) for ``CART``
+3. `eccentricity` for ``COM`` or ``KEP``, `y` (au) for ``CART``
+4. `inclination` (deg) for ``COM`` or ``KEP``, `z` (au) for ``CART``
+5. `longitude of the ascending node` (deg) for ``COM`` and ``KEP``,
+   `dx/dt` (au/day) for ``CART``
+6. `argument of the periapsis` (deg) for ``COM`` and ``KEP``, `dy/dt`
+   (au/day) for ``CARqT``
+7. `epoch of perihelion` (modified Julian date) for ``COM``, mean
+   anomaly (deg) for ``KEP``, `dz/dt` for ``CART``
+8. `orbital element type`; integer value: ``CART``: 1, ``COM``: 2,
+   ``KEP``: 3
+9. `epoch` of the osculating elements (modified Julian date)
+10. `timescale type` of the epochs provided; integer value: ``UTC``:
+    1, ``UT1``: 2, ``TT``: 3, ``TAI``: 4
+11. `absolute magnitude` (``KEP`` or ``CART``) or `M1` parameter
+    (``COM``)
+12. `photometric slope parameter` (``KEP`` or ``CART``) or `K1`
+    parameter (``COM``)
+
+In order to provide compatibility with the underlying FORTRAN library,
+only use double values (``dtype=np.double``) and set the corresponding
+order for the array (``order='F'``). The following lines provide an
+example orbit array defining three individual orbits using the
+Keplerian orbit definition:
+
+    >>> import numpy as np
+    >>> orbits = np.array(
+    ...    np.array([[0, 1, 2],  # id
+    ...              [1.46905, 2.49241, 2.43064],  # a
+    ...              [0.33435, 0.78954, 0.70177],  # e
+    ...              np.deg2rad([14.3024, 7.99749, 0.52734]),  # incl
+    ...              np.deg2rad([224.513, 124.587, 334.990]),  # longnode
+    ...              np.deg2rad([27.5419, 267.104, 205.376]),  # argper
+    ...              np.deg2rad([324.697, 297.655, 5.56391]),  # mean anom
+    ...              [3, 3, 3],  # orbit type
+    ...              [51544.5, 51544.5, 51544.5],  # epoch
+    ...              [1, 1, 1],  # timescale type
+    ...              [12.5, 6.3, 20.5],  # absolute magnitude
+    ...              [0.15, 0.15, 0.15]]).transpose(),  # slope parameter
+    ...    dtype=np.double, order='F')
+
+Note that angles have to be provided in radians, not in degrees;
+epochs have to be provided as modified Julian dates. Also note that
+the inner array is transposed; orbit arrays consist of a single array
+(a single orbit) or a nested array of single orbit arrays.
+
+
+Initializing pyoorb
+^^^^^^^^^^^^^^^^^^^
+
+Before any pyoorb functionality can be used, the module has to be
+initialized using the following lines:
+
+    >>> import os
+    >>> import pyoorb as oo
+    >>> ephfile = os.path.join(os.getenv('OORB_DATA'), 'de430.dat')
+    >>> oo.pyoorb.oorb_init(ephfile)
+
+Note that this requires the ``'OORB_DATA'`` environment variable to be
+properly defined (see installation guide above).
```

Comment:
May be useful to comment that the ephfile could be a different ephemeris file (not necessarily de430.dat, could use de405.dat if desired for compatibility with older predictions, etc.).

Reply:
done